packages/babel-compat-data/data/plugins.json

```diff
@@ -275,7 +285,7 @@
   },
   "transform-block-scoping": {
     "chrome": "49",
-    "edge": "14",
+    "edge": "79",
```

Comment:
I should check why this changed

Reply:
I'm guessing https://github.com/kangax/compat-table/pull/1569?

Reply:
That commit didn't change the data files :thinking:

Reply:
We use essentially the same code as compat-table's HTML generation to interpolate versions via `environments.json`. As I said, was just a guess... I'll try to dig a bit further when I get home.

Reply:
It's because of [`equals` field](https://github.com/kangax/compat-table/pull/1569/files#diff-ca0f8d6af0c57ea63bddc9086afc0bc2R2214). Should be correct.

Reply:
Well, `edge14` still supports `let`/`const`. Maybe it's  bug in our script?

Reply:
https://kangax.github.io/compat-table/es6/ has the same bug. Any version of Edge before 79 is marked as not supporting `let`/`const`.